docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,51 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+List calls for nested children under the resource group segment is allowed only if parent resource under the resource group exist.
```

Comment:
When the parent resource is scoped under a resource group, the path for the list calls of its child resources must also be scoped under a resource group